### DISCUSSION TITLE
resolutions are not respected

### DISCUSSION THREAD

**Discussion by sasial-dev** [renovatebot/renovate#10793] [Request Help]
The resolutions in my package.json are not respected with renovate.
Should I create an issue for this?

**Comment by rarkins** [comment]
No, please first describe here in much more detail what you mean, and create a [minimal reproduction repo](https://github.com/renovatebot/renovate/blob/main/docs/development/minimal-reproductions.md) to demonstrate it. Then we'll confirm if it's a bug, feature, or other.
